File: FiveRockingFingers/FRF.Core/Models/PricingTerm.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FRF.Core.Models
{
    public class PricingTerm
    {
        public string Product { get; set; }
        public string Sku { get; set; }
        public string Term { get; set; }
        public List<PricingDimension> PricingDimensions { get; set; }
        public string LeaseContractLength { get; set; }
        public string OfferingClass { get; set; }
        public string PurchaseOption { get; set; }
    }
}

File: FiveRockingFingers/FRF.Web/ClientApp/src/interfaces/PricingTerm.ts
import PricingDimension from './PricingDimension';

export default interface PricingTerm {
    product: string;
    sku: string;
    term: string;
    pricingDimensions: PricingDimension[];
    leaseContractLength: string;
    offeringClass: string;
    purchaseOption: string;
}